diff --git i/src/vs/workbench/contrib/chat/browser/chat.shared.contribution.ts w/src/vs/workbench/contrib/chat/browser/chat.shared.contribution.ts
index 14210b54..c1bf67ce 100644
--- i/src/vs/workbench/contrib/chat/browser/chat.shared.contribution.ts
+++ w/src/vs/workbench/contrib/chat/browser/chat.shared.contribution.ts
@@ -2239,7 +2239,7 @@ configurationRegistry.registerConfiguration({
 		[ChatAIDisabledSettingId]: {
 			type: 'boolean',
 			description: nls.localize('chat.disableAIFeatures', "Disable and hide built-in AI features provided by GitHub Copilot, including chat and inline suggestions."),
-			default: false,
+			default: true,
 			scope: ConfigurationScope.WINDOW,
 		},
 		[ChatConfiguration.TitleBarSignInEnabled]: {
diff --git i/src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.contribution.ts w/src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.contribution.ts
index 067580a4..dfa4bb83 100644
--- i/src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.contribution.ts
+++ w/src/vs/workbench/contrib/welcomeGettingStarted/browser/gettingStarted.contribution.ts
@@ -327,7 +327,7 @@ configurationRegistry.registerConfiguration({
 				localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'workbench.startupEditor.terminal' }, "Open a new terminal in the editor area."),
 				localize({ comment: ['This is the description for a setting. Values surrounded by single quotes are not to be translated.'], key: 'workbench.startupEditor.agentSessionsWelcomePage' }, "Open the Agent Sessions Welcome page. Will override the workbench secondary side bar visibility settings."),
 			],
-			'default': 'welcomePage',
+			'default': 'none',
 			'description': localize('workbench.startupEditor', "Controls which editor is shown at startup, if none are restored from the previous session."),
 			'experiment': { mode: 'auto' },
 			agentsWindow: { default: 'none', readOnly: true },
